main.go

```diff
@@ -102,23 +135,35 @@ func filterVariants(module, variant string, variantsMap gradle.Variants) (gradle
 		variantsMap = gradle.Variants{module: v}
 	}
 
-	// if variant not set: use all variants
+	// if variant not set: use all variants, except utility ones
 	if variant == "" {
+		for module, variants := range variantsMap {
+			variantsMap[module] = filterUtilityVariants(variants)
```

Comment:
Feels scary to modify a map while iterating over it. Even if we only change the values. Does go handle this nicely?

Reply:
It is legal, adding/removing elements is more tricky tough. https://github.com/golang/go/issues/9926